client/src/index.js

```diff
@@ -3,5 +3,13 @@ import ReactDOM from 'react-dom';
 
 import './index.css';
 import App from './App';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
 
```

Comment:
you can remove the Route and Link in the imports here as you are not using those components in this file, only the BrowserRouter as Router part is needed on this one